examples/blog/app.py

```diff
@@ -195,10 +195,13 @@ def index():
 def create():
     if request.method == 'POST':
         if request.form.get('title') and request.form.get('content'):
-            entry = Entry.create(
-                title=request.form['title'],
-                content=request.form['content'],
-                published=request.form.get('published') or False)
+            try:
+                entry = Entry.create(
+                    title=request.form['title'],
+                    content=request.form['content'],
+                    published=request.form.get('published') or False)
+            except IntegrityError:
+                flash('That title already exists.', 'warning')
             flash('Entry created successfully.', 'success')
             if entry.published:
```

Comment:
Assuming the integrity error was raised, what do you think is going to happen here?